docs/en/api/lynx-api/lynx/lynx-global-props.mdx

```diff
@@ -8,7 +8,14 @@
 
 ## Usage
 
-```jsx
+```tsx
+declare module '@lynx-js/types' {
```

Comment:
Could you pre-append a sentence of comment?

Reply:
Inside the code block or outside?

Reply:
I believe it makes more sense to include it internally, as comments are just a part of the code.